packages/project/test/workflow.test.ts

```diff
@@ -205,3 +206,61 @@ test('map uuids to ids', (t) => {
   t.deepEqual(w.index.id[uuid_ac], 'a-c');
   t.deepEqual(w.index.id[uuid_bc], 'b-c');
 });
+
+test('canMergeInto: empty histories', (t) => {
+  const sourceWf = generateWorkflow('trigger-x');
+  const targetWf = generateWorkflow('trigger-y');
+  t.deepEqual(sourceWf.history, []);
+  t.deepEqual(targetWf.history, []);
+  t.true(sourceWf.canMergeInto(targetWf)); // allowed
```

Comment:
Can we add a line break on this final assertion? It's the most important bit of the test and I want it to stand alone a bit more
```
  const sourceWf = generateWorkflow('trigger-x');
  const targetWf = generateWorkflow('trigger-y');
  t.deepEqual(sourceWf.history, []);
  t.deepEqual(targetWf.history, []);

  t.true(sourceWf.canMergeInto(targetWf)); 
```